pages/employees.tsx

```diff
@@ -152,6 +148,45 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     },
   });
 
+  let employees = null;
+  if (dbEmployees) {
+    employees = dbEmployees.map((employee) => {
+      return {
+        ...employee,
+        createdAt: DateTime.fromJSDate(employee.createdAt).toFormat(
+          "MM/dd/yy hh:mm:ss a"
+        ),
+        updatedAt: DateTime.fromJSDate(employee.updatedAt).toFormat(
+          "MM/dd/yy hh:mm:ss a"
+        ),
+        hireDate: DateTime.fromJSDate(employee.hireDate).toFormat("MM/dd/yyyy"),
+        endEmploymentDate: employee.endEmploymentDate
+          ? DateTime.fromJSDate(employee.endEmploymentDate).toFormat(
+              "MM/dd/yyyy"
+            )
+          : null,
+        manager: {
+          ...employee.manager,
+          createdAt: employee.manager
+            ? DateTime.fromJSDate(employee.manager.createdAt).toFormat(
+                "MM/dd/yy hh:mm:ss a"
+              )
+            : null,
+          updatedAt: employee.manager
+            ? DateTime.fromJSDate(employee.manager.updatedAt).toFormat(
+                "MM/dd/yy hh:mm:ss a"
+              )
+            : null,
+          hireDate: employee.manager
+            ? DateTime.fromJSDate(employee.manager.hireDate).toFormat(
+                "MM/dd/yyyy"
+              )
+            : null,
+        },
+      };
+    });
+  }
+
```

Comment:
Do we use all of these properties in the view? If not make a new type that only has the data the component needs

Reply:
Added and reshaped type